tasks/release.yml

```diff
@@ -0,0 +1,15 @@
+# TODO These tasks should be considered for addition to the ansible
+#      rhsm_repositories module, which would remove the need to set facts to
+#      ensure idempotence when using the command module.
+- name: Unset subscription-manager release
+  command: subscription-manager release --unset
+  when: rhsm_release_unset and oasis_roles_rhsm_release
+  register: rhsm_release_unset_task
```

Comment:
On this task and the next, is there any way to determine if changes were made? Sometimes there's a way to set `changed_when` based on the stdout of the command or such?

Reply:
Sadly, no. Here's the output I'm working with:
```
# subscription-manager release --show
Release: 7.0
# subscription-manager release --set 7.0
Release set to: 7.0
# subscription-manager release --unset
Release preference has been unset
# sudo subscription-manager release --unset
Release preference has been unset
```

So you can see we still get the "Release set to" text, even when the release was already set to the "new" value. The same is true when using `unset`; s-m reports that a change was made, even when that change is to the current value.

Reply:
I suggest that we embellish the comment a bit to explain the unfortunate behavior here.  Sounds like getting this into the module itself is the real path forward.  Until then we can have some info to explain why we can't track actual changes here.

Reply:
I'm not sure what you're asking for. The unfortunate behavior is that we have to set facts internally to ensure idempotence, and the fix is to put the behavior into the rhsm_repository module. That's what the comment already says.

I don't think it's worthwhile to write a 20-line comment in the role to explain this; I was planning to open up a github issue spelling out the problem fully once I actually had code in this repo to link to.

Reply:
I was asking for info in the code comments about the subscription manager output, similar to your earlier comment in this thread.  As someone who hasn't done much with s-m, this seemed useful to me.  However, a GitHub issue with that info would also suffice.

Reply:
Cool. I'll make sure to reference this comment in the issue and see about providing a more complete picture of how s-m is working for us (and against us) here, and how we can get this taken care of upstream.